sos/report/plugins/ceph_common.py

```diff
@@ -118,4 +118,11 @@ def setup(self):
             "ceph -v",
         ])
 
+    def postproc(self):
+        protect_keys = [
+            "rgw keystone admin password",
+        ]
+        regex = fr"(^({'|'.join(protect_keys)})\s*=\s*)(.*)"
```

Comment:
I dont understand the '|'.join part. Since the list contains just one item, the `'|'.join(..)` will be pure `'rgw keystone admin password'` - is this a preparation for possible future keys to obfuscate, i.e. for protect_keys to be a multi-item list?

Reply:
yeah, that was the idea, that if there is anything in the future, it's just a one-line change.

Can remove that, and make it just simple, and only obfuscate that key for the time being

Reply:
No prob having it as is, thanks for explanation.